blockstore/splitstore/README.md

```diff
@@ -27,9 +27,34 @@ If you intend to use the discard coldstore, your also need to add the following:
     ColdStoreType = "discard"
 ```
 In general you _should not_ have to use the discard store, unless you
-are running a network booster or have very constrained hardware with
-not enough disk space to maintain a coldstore, even with garbage
-collection.
+are running a network assistive node (like a bootstrapper or booster)
+or have very constrained hardware with not enough disk space to
+maintain a coldstore, even with garbage collection. It is also appropriate
+for small nodes that are simply watching the chain.
+
+## Configuration Options
+
+These are options in the `[Chainstore.Splitstore]` section of the configuration:
+
+- `HotStoreType` -- specifies the type of hotstore to use.
+  The only currently supported option is `"badger"`.
+- `ColdStoreType` -- specifies the type of coldstore to use.
+  The default value is `"universal"`, which will use the initial monolith blockstore
+  as the coldstore.
+  The other possible value is `"discard"`, as outlined above, which is specialized for
+  running without a coldstore. Note that the discard store wraps the initial monolith
+  blockstore and discards writes; this is necessary to support syncing from a snapshot.
+- `MarkSetType` -- specifies the type of markset to use during compaction.
```

Comment:
Have we defined what a "markset" is? If not, it makes this hard to follow.

Reply:
Sure, I'll define it.

Reply:
Added a definition.